src/services/string.ts

```diff
@@ -0,0 +1,166 @@
+/**
+ * This file is part of Threema Web.
+ *
+ * Threema Web is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Threema Web. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+export class StringService implements threema.StringService {
+
+    /**
+     * Split a string into chunks
+     * @param str the string to chunk
+     * @param length max length of a chuck
+     * @param offset searching for a separator in the given offset, 0 disable separator indexing
+     */
+    public chunk(str: string, length: number, offset: number = 10): string[] {
```

Comment:
Could it be that this isn't actually being used right now? The submitText() method uses `byteChunk`.